src/helpers.ts

```diff
@@ -102,10 +102,11 @@ export const renderMarkdown = (markdown: string): string => {
         // Allow highligh.js styles, e.g.
         // <span class="hljs-keyword">
         // <code class="language-javascript">
-        allowedTags: [...sanitize.defaults.allowedTags, 'span'],
+        allowedTags: [...sanitize.defaults.allowedTags, 'span', 'h4', 'a', 'p', 'strong', 'details', 'summary'],
```

Comment:
```suggestion
        allowedTags: [...sanitize.defaults.allowedTags, 'span', 'details', 'summary'],
```
`h4`, `p`, `strong` are all already in `defaults.allowedTags`: https://sourcegraph.com/github.com/punkave/sanitize-html/-/blob/src/index.js#L572